hidden_walkthroughs/4-protecting-apis/walkthrough.adoc

```diff
@@ -7,7 +7,7 @@
 :route: https://wt4-{user-username}-3scale.{openshift-app-host}
 
 //attributes
-:title: 4 - Protecting APIs using 3scale API Management Platform
+:title: 4 - Protecting APIs using 3scale API Management Platform - testing
```

Comment:
@tonyxrmdavidson Same as other comment. I think this should be removed so it doesn't show up in the UI.

Reply:
These should all be sorted now.